lib/go_to_webinar.rb

```diff
@@ -9,37 +9,37 @@
 
 module GoToWebinar
   class Client
-    
+
     include HTTParty
     format :json
 
     include GoToWebinar::Webinars
     include GoToWebinar::Registrants
     include GoToWebinar::Attendees
     include GoToWebinar::Sessions
-    
+
     attr_accessor :access_token
     attr_accessor :organizer_key
-    
+
     def initialize(access_token, organizer_key, extra_params = {})
-      
-      # the access token from oauth 
+
+      # the access token from oauth
       @access_token = access_token
       @organizer_key = organizer_key
-      
+
       @default_params = {
-        :base_uri => "https://api.citrixonline.com/G2W/rest/organizers/#{@organizer_key}/",
+        :base_uri => "https://api.getgo.com/G2W/rest/organizers/#{@organizer_key}/",
```

Comment:
Couldn't this be using the URL from the gem? Or is that not accessible?

Reply:
you mean GoToWebinar-Ruby gem using the URL from omniauth-gotowebinar gem ?

Reply:
It's not a dependency. Ignore me.